packages/devtools/test/integration_tests/debugger_test.dart

```diff
@@ -155,132 +145,151 @@ void debuggingTests() {
+      final String source = appFixture.scriptSource;
+      final int exceptionLine = CliAppFixture.parseExceptionLines(source).first;
 
-    // verify location
-    expect(
-      await debuggingManager.getLocation(),
-      endsWith('${appFixture.appScriptPath}:$exceptionLine'),
-    );
+      final DevtoolsManager tools =
+          DevtoolsManager(tabInstance, webdevFixture.baseUri);
+      await tools.start(appFixture);
+      await tools.switchPage('debugger');
 
-    // verify locals, including the exception object
-    expect(await debuggingManager.getVariables(), [
-      '<exception>:StateError',
-      'foo:2',
-    ]);
+      final String currentPageId = await tools.currentPageId();
+      expect(currentPageId, 'debugger');
 
-    // resume
-    await debuggingManager.setExceptionPauseMode('Unhandled');
-    await debuggingManager.resume();
+      final DebuggingManager debuggingManager = DebuggingManager(tools);
 
-    // verify state resumed
-    expect(await debuggingManager.getState(), 'running');
-  });
+      // verify running state
+      expect(await debuggingManager.getState(), 'running');
 
-  test('console output', () async {
-    appFixture = await CliAppFixture.create(
-        'test/fixtures/debugging_app_exception.dart');
+      // set break on exceptions mode
+      await debuggingManager.setExceptionPauseMode('All');
 
-    final DevtoolsManager tools =
-        DevtoolsManager(tabInstance, webdevFixture.baseUri);
-    await tools.start(appFixture);
-    await tools.switchPage('debugger');
+      // wait for paused state
+      await waitFor(() async => await debuggingManager.getState() == 'paused');
 
-    final String currentPageId = await tools.currentPageId();
-    expect(currentPageId, 'debugger');
+      await shortDelay();
 
-    final DebuggingManager debuggingManager = DebuggingManager(tools);
+      // verify location
+      expect(
+        await debuggingManager.getLocation(),
+        endsWith('${appFixture.appScriptPath}:$exceptionLine'),
+      );
 
-    // verify running state
-    expect(await debuggingManager.getState(), 'running');
+      // verify locals, including the exception object
+      expect(await debuggingManager.getVariables(), [
+        '<exception>:StateError',
+        'foo:2',
+      ]);
 
-    // wait until there's console output
-    await waitFor(
-        () async => (await debuggingManager.getConsoleContents()).isNotEmpty);
+      // resume
+      await debuggingManager.setExceptionPauseMode('Unhandled');
+      await debuggingManager.resume();
 
-    // verify the console contents
-    expect(
-      await debuggingManager.getConsoleContents(),
-      contains('1\n'),
-    );
-  });
+      // verify state resumed
+      expect(await debuggingManager.getState(), 'running');
+    });
+
+    test('console output', () async {
+      appFixture = await CliAppFixture.create(
+          'test/fixtures/debugging_app_exception.dart');
+
+      final DevtoolsManager tools =
```

Comment:
this diff is quite hard to read. Are there any real changes here?

Reply:
Are you looking at the PR, or the diff from the link in my comment above? Hopefully the link in the comment will give you a clean diff :-)